spec.md

```diff
@@ -284,6 +314,48 @@ how platforms might expose these values to their users.
           "5 TB storage",
           "40 concurrent connections"
         ]
+      },
+      "schemas": {
+        "service_instance": {
+          "create": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "properties": {
+                "billing-account": {
+                  "description": "Billing account number used to charge use of shared fake server.",
```

Comment:
Do we want (or need) to place any requirements on the set of fields used to describe the properties?  I'm guessing no but wanted to ask to be sure.